notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2663 @@
+{
```

Comment:
<p>Is there specific documentation from stella that is relevant to point people to for the hint?</p><p> </p><p>May be good to clarify if 6 should be done in addition to 5, or separately from 5.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>

Reply:
<p>Also I think after reviewing the solutions notebook, there could probably be more detail given for 7. I would suggest breaking down what this means into specific tasks, maybe one of which is like "Estimate the uncertainty in the period at max power by... [something something]"</p>